app/views/analyses/new/_form.html.haml

```diff
@@ -0,0 +1,93 @@
+.container
+  %h3.section-heading
+    #{t("analysis.analysis_type.#{@analysis.analysis_type}")} analysis
+
+  %div{ class: "new-#{@analysis.analysis_type}-analysis" }
+    = render partial: '/errors', locals: { errors: @form.errors }
+
+    = fields_for(@form, builder: BaseFormBuilder) do |f|
+      .form-group
+        = f.text_field :name, required: true, label: 'Analysis name',
+          help: 'Please, provide <b>name</b> for the analysis.', form: 'new_analysis'
+
+      .form-group
+        = label_tag :genotype_data_file, 'Genotype data', class: 'required'
+        = render partial: "/analyses/new/genotype_data_file_help"
+
+        .genotype-data-file-errors.errors.hidden
+
+        = form_tag analyses_data_files_path, multipart: true, method: :post do
+          = hidden_field_tag "analysis_data_file[data_type]", "gwas_genotype", id: "genotype-data-file-data-type"
+
+          .genotype-data-file-fileinput.fileinput{ class: "#{'hidden' if @form.genotype_data_file}" }
+            .btn.btn-primary.fileinput-button
+              %label{ for: "genotype-data-file" } Choose genotype file...
+              = file_field_tag "analysis_data_file[file]", class: 'genotype-data-file', id: 'genotype-data-file'
+
+            %small.help-block
+              Please upload the genotype VCF or CSV file here. Max file size is 50 MiB.
+
+        .genotype-data-file{ class: "#{'hidden' unless @form.genotype_data_file}" }
+          %span.file-name= @form.genotype_data_file.try(:file_file_name)
+
+          = delete_analysis_data_file_button(@form.genotype_data_file,
+            class: 'btn btn-default delete-genotype-data-file')
+
+      .form-group{ class: (:hidden unless @form.map_data_file) }
```

Comment:
You use the traditional string class name notation several lines back, for the same purpose. Please use the same here for consistency.